src/main/java/christmas/util/Converter.java

```diff
@@ -0,0 +1,44 @@
+package christmas.util;
+
+import christmas.exception.ErrorMessage;
+import christmas.exception.InvalidDateException;
+import christmas.global.Const;
+import java.util.Arrays;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class Converter {
+    private static final String SEPARATOR = ",";
+    private static final String PAIR_SEPARATOR = "-";
+
+    public static Map<String, Integer> convertToPair(String userMenus) {
+        Map<String, Integer> resultMap = Arrays.stream(userMenus.split(SEPARATOR))
+                .map(Converter::removeWhiteSpace)
+                .map(menu -> menu.split(PAIR_SEPARATOR))
+                .collect(Collectors.toMap(
+                        pair -> pair[Const.START_VALUE],
+                        pair -> parseToInt(pair[Const.INCREMENT]),
+                        (existedInput, reInput) -> Validator.validateDuplication(existedInput)
+                ));
```

Comment:
(버그에요)
```
샴페인-1,초코케이크
```
위 입력 값을 넣으면 배열의 원소를 벗어난 에러가 발생합니다.
그 이유는 - 기준으로 분할 할 때 문자 `-`가 존재하지 않는 경우 첫 번째 원소가 존재하지 않기 떄문입니다.
```suggestion
                .map(menu -> menu.split(PAIR_SEPARATOR))
                .peek(pair -> pair 사이즈가 2일 때 에러 발생)
                .collect(Collectors.toMap(
                        pair -> pair[Const.START_VALUE],
                        pair -> parseToInt(pair[Const.INCREMENT]),
                        (existedInput, reInput) -> Validator.validateDuplication(existedInput)
                ));
```
위와 같이 작성해주면 문제를 해결할 수 있습니다!